src/main/resources/application-local.yml

```diff
@@ -8,7 +8,7 @@ localstack:
 aws:
   sqs:
     queue:
-      url: ${localstack.base-url}/queue/case-creator-queue
+      url: ${localstack.base-url}/queue/migration-queue
```

Comment:
I disagree with this change, the queue should be case-creator, there is and should be only a singular queue in the localstack.

Reply:
The default local setup for hocs-cms-data-extractor puts it on the migration queue, so one side needed to change, but happy for it to be the LHS if you prefer?

Reply:
For a week I suppose this is fine, could we both make a note to revert this after?

Reply:
Optionally override it? https://github.com/UKHomeOffice/hocs-case-creator/pull/286/files/3cd2274bdc578ccabccbfc5a6e00c150ea2878f1#r1230732425